compiler/luci-interpreter/pal/cmsisnn/PALreference_ops.h

```diff
@@ -0,0 +1,1568 @@
+/*
```

Comment:
You may want to use a relative symlink to avoid code duplication (git allows it, but IDK if the repo policy does)

Reply:
NVM, the file contents are different.